### ISSUE TITLE
Add Captcha system to prevent bots

### ISSUE THREAD

**Issue by alexandrevicenzi** [SUSE/hackweek#1622]
[Devise allows Google reCaptcha](https://github.com/heartcombo/devise/wiki/How-To%3A-Use-Recaptcha-with-Devise) and I wonder why this is not enabled in the website to (attempt to) prevent bots from spamming Hack Week website.

It seems fairly easy to integrate, any reasons why we don't have it yet?

**Comment by hennevogel** [comment]
> It seems fairly easy to integrate, any reasons why we don't have it yet?

Because this problem is not exclusive to this web application but to all behind the IDP. And because this is not an option for some of the web apps. And... as usually, it's complicated...

This is an open source project, if you want to contribute this, I'd be happy to integrate it 😃

**Comment by alexandrevicenzi** [comment]
Yes, true, the Hack Week website is not the only one to receive spam, but it also offers no content moderation.

On a short internal discussion, it seems that some are against Captcha because it is an extra step during login. I'm not sure if it would be worth having it or not.
